admin/client/dist/js/MemberDatetimeOptionsetField.js

```diff
@@ -32,9 +32,9 @@
 
 		$('.memberdatetimeoptionset .toggle').entwine({
 			onclick: function onclick(e) {
-				jQuery(this).closest('.form__field-description').parent().find('.toggle-content').toggle();
+				jQuery(this).closest('.description').parent().find('.toggle-content').toggle();
```

Comment:
We're currently not minifying/mangling dist files because it impedes our ability to set breakpoints - some combination of babel/browserify/chrome means that we can't do that on minified files. This should be fixed by the planned switch to Webpack, which will also remove extraneous dist files like the one discussed here - see https://github.com/silverstripe/silverstripe-framework/pull/5918